docs/guides/postgres/branch/customization/index.md

```
@@ -0,0 +1,554 @@
+$ kubectl wait --for=condition=Ready branch/dev-branch-ha -n demo --timeout=10m
+branch.courier.kubedb.com/dev-branch-ha condition met
+
+$ kubectl get branch -n demo dev-branch-ha
+NAME            PHASE   MODE    TARGET            FRESHNESS   AGE
+dev-branch-ha   Ready   Local   dev-postgres-ha   1s          81s
+```
+
+`status.snapshot.members` has one entry per source volume:
+
+```bash
+$ kubectl get branch -n demo dev-branch-ha -o jsonpath='{.status.snapshot}'
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)